site/layouts/partials/docs-sidebar.html

```diff
@@ -34,7 +35,7 @@
           {{ if eq $group_slug "migrations" }}
             {{- $href = urls.JoinPath "/docs" $.Site.Params.docs_version $doc_slug "/" }}
           {{ end }}
-          <li><a href="{{ $href }}" class="bd-links-link d-inline-block{{ if $is_active }} active{{ end }}"{{ if $is_active }} aria-current="page"{{ end }}>{{ $doc.title }}</a></li>
+          <li><a href="{{ $href }}" class="bd-links-link d-inline-block{{ if $is_active }} active{{ end }}{{if $is_draft }} text-decoration-line-through{{ end }}"{{ if $is_active }} aria-current="page"{{ end }}>{{ $doc.title }}</a></li>
```

Comment:
I think that `<del>` is not accurate, and `<s>`, why not... 
But a `visually-hidden` mention is a solution I often like because it always works 👍  So go for it!?

Reply:
Yep, let's start with a `visually-hidden` 👌